Automattic/wp-calypso
# Map Block: Alignment

#### Changes proposed in this Pull Request

Somehow block alignment got lost. This PR brings it back.

#### Testing instructions

- Spin up JN instance: https://jurassic.ninja/create?wordpress-5-beta&gutenpack&calypsobranch=fix/map-alignment
- Enable Jetpack
- Be sure you are using a theme that supports full and wide alignment, such as `Twentynineteen`
- Add a map block. 
- Try all three alignments, saving and previewing (or publishing) after each. Verify that the alignment is respected in view.

Fixes #

Partially resolves https://github.com/Automattic/wp-calypso/issues/28782
